scripts/install_cairo_bindings.sh

```diff
@@ -5,37 +5,33 @@ set -e
 # clean up
 if [ "$1" == "--cleanup" ]; then
   poetry env info -p | xargs rm -rf
-  if [[ $(uname -m) == 'arm64' ]]; then
+  if [[ $(uname -s) == 'Darwin' ]]; then
     CFLAGS=-I/opt/homebrew/opt/gmp/include LDFLAGS=-L/opt/homebrew/opt/gmp/lib poetry install
   else
     poetry install
   fi
 fi
 
 function install_dev() {
-  git pull --recurse-submodules
+  git pull origin $(git rev-parse --abbrev-ref HEAD) --ff-only --recurse-submodules
 
-  pushd cairo
-  pushd crates/cairo-lang-python-bindings
+  pushd cairo/crates/cairo-lang-python-bindings
   rustup override set nightly-2022-11-03 || return 1;
   maturin develop --release || return 1;
-  popd # crates/cairo-lang-python-bindings
-  popd # cairo
+  popd
 }
 
 function install_prod() {
-  git pull --recurse-submodules
+  git pull origin $(git rev-parse --abbrev-ref HEAD) --ff-only --recurse-submodules
 
-  pushd cairo
-  pushd crates/cairo-lang-python-bindings
+  pushd cairo/crates/cairo-lang-python-bindings
   rustup override set nightly-2022-11-03 || return 1;
   maturin build || return 1;
-  popd # crates/cairo-lang-python-bindings
+  popd
```

Comment:
```suggestion
  popd # cairo/crates/cairo-lang-python-bindings
```
makes it easier to read